crates/sui-core/src/consensus_adapter.rs

```diff
@@ -49,13 +49,54 @@ type SerializedTransactionInfoResponse = Vec<u8>;
 
 /// Channel to notify the caller when the Sui certificate has been sequenced.
 type TxSequencedNotifier = oneshot::Sender<SuiResult<SerializedTransactionInfoResponse>>;
+type TxSequencedNotifierClose = oneshot::Sender<()>;
 
 /// Message to notify the consensus listener that a new transaction has been sent to consensus
 /// or that the caller timed out on a specific transaction.
 #[derive(Debug)]
 pub enum ConsensusListenerMessage {
```

Comment:
Shouldn't this be a simple struct?

Reply:
Yeah, but the only thing we would do with the structure is break it up again, so I am skipping it right now.